moveit_setup_assistant/package.xml

```diff
@@ -16,26 +16,33 @@
   <url type="bugtracker">https://github.com/ros-planning/moveit2/issues</url>
   <url type="repository">https://github.com/ros-planning/moveit2</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
I can't comment on lines above this in this file, but please update the version of this package to match the rest of moveit2.  The current version is 2.3.0.

Also, please update this to use package format 3 as that is the new standard for package.xml files.  Reference: https://www.ros.org/reps/rep-0149.html